grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -565,7 +564,7 @@ class ProductAvailabilityService {
 
 
     void updateProductAvailability(Location location) {
-        def isBinLocation = location?.locationType?.locationTypeCode in [LocationTypeCode.BIN_LOCATION, LocationTypeCode.INTERNAL]
+        def isBinLocation = location?.isInternalLocation()
```

Comment:
It's ok if you already made these changes. We should just add some notes in the ticket to ensure we test them.